### PR TITLE
feat: cron

### PR DESCRIPTION
**Pull request by christopherbrumm**
This PR adds cron support for the `sync` command. Therefore, the interval is replaced with a cron interval using https://github.com/go-co-op/gocron.

### CODE REVIEW

File: cmd/dlt/commands/sync.go [hunk 1]
```diff
@@ -57,32 +57,78 @@ var syncCmd = &cobra.Command{
...
 		sigCount := 0
 
+		cronScheduler, err := gocron.NewScheduler()
+		if err != nil {
+			logger.Error().Str("err", err.Error()).Msg("failed to create cron scheduler")
+			return
+		}
+
+		for _, c := range connections {
+			loader, err := l.SetupLoader(configPath, c.Name, false, fromBundleId, math.MaxInt64, force)
```
Comment (mbreithecker) [inline cmd/dlt/commands/sync.go:91]: Could it be dangerous that we reuse the same loader again for every iteration? e.g. somebody adds a state variable to the loader one day, which does not properly reset after the run is finished

**Comment by mbreithecker** [comment]
Counter does not reset on continuous sync: 
3:53PM INF [DLT] Finished sync for kyve! Took 479 seconds module=cmd

Omit these messages in sync:
3:54PM WRN [DLT] found loaded data in destination connection=celestia highest_bundle_id=71710 module=loader
3:54PM INF [DLT] set new from_bundle_id - this step can be skipped with --force connection=celestia id=71711 module=loader

File: cmd/dlt/commands/sync.go (hunk 1 above)
Comment (christopherbrumm) [reply cmd/dlt/commands/sync.go:91]: A new loader is created every iteration in order to add it to the `gocron.NewTask`. Right now, this is not dangerous, but I will add a comment to keep it in mind for potential future changes.

File: cmd/dlt/commands/connections.go [hunk 2]
```diff
@@ -101,16 +103,17 @@ var connectionsListCmd = &cobra.Command{
 
 		if len(config.Connections) > 0 {
 			columnOffset := 2
-			maxNameLen, maxSourceLen, maxDestinationLen := len("Name"), len("Source"), len("Destination")
+			maxNameLen, maxSourceLen, maxDestinationLen, maxCronLen := len("Name"), len("Source"), len("Destination"), len("Destination")
```
Comment (mbreithecker) [inline cmd/dlt/commands/connections.go:106]: Should the last argument be len("Cron") or something like that?

Comment (christopherbrumm) [reply cmd/dlt/commands/connections.go:106]: Yes, good catch!

**Review by mbreithecker — APPROVED** [review]

**Pull request merged**